### PR TITLE
All the necessary changes to add the visualize all feature

### LINKED ISSUE #48: Adding new feature Visualize_all
I have been working on this since 2 days, it's almost done, but still needs  more tests, it would be really helpful to work in this feature if we got some of the pull requests merged

### PR DESCRIPTION
**Pull request by ghassanmas**
Relates and Closes #48

### CODE REVIEW

File: public/js/all.js [hunk 1]
```diff
@@ -0,0 +1,242 @@
+/*eslint-disable*/
+document.getElementById('next').addEventListener('click', next);
+document.getElementById('back').addEventListener('click', back);
+document.getElementById('play').addEventListener('click', play);
+document.onkeydown = checkKey;
+
+const urlParams = new URLSearchParams(window.location.search);
```
Comment (Karyum) [inline public/js/all.js]: ES6 on the frontend

File: public/js/all.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+const url = '/visualize_all_json';
+var i = 0;
+var size;
+var len;
+var simulation;
+var result;
+var group;
```
Comment (Karyum) [inline public/js/all.js]: these var can be on the same line for less lines check if it works after it

File: public/js/all.js (hunk 1 above)
Comment (ghassanmas) [reply public/js/all.js]: done

File: public/js/all.js (hunk 2 above)
Comment (ghassanmas) [reply public/js/all.js]: done

**Pull request merged**